internal/firefox.py

```diff
@@ -146,8 +147,8 @@ def launch(self, job, task):
             self.extension_id = self.addons.install(extension_path, temp=True)
             logging.debug('Resizing browser to %dx%d', task['width'], task['height'])
             self.marionette.set_window_rect(x=0, y=0, height=task['height'], width=task['width'])
-            if 'browserVersion' in self.marionette.session_capabilities:
-                self.browser_version = self.marionette.session_capabilities['browserVersion']
+            self.browser_version = self.marionette.session_capabilities.get['browserVersion']
+            self.browser_build = self.marionette.session_capabilities.get['moz:buildID']
```

Comment:
This will fail because `get` is not a dict but a method on the dict. You will need to use round brackets, such as:

```
self.browser_version = self.marionette.session_capabilities.get('browserVersion')
```

Reply:
Fixed.